src/redux/Books/books.js

```diff
@@ -0,0 +1,17 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```

Comment:
- [ ] These action types need to follow the duck's pattern, according to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/166410186-2c3225d6-4dc0-4db7-baf9-cbf0a7e2ba45.png)


Hint:  Examples of duck's pattern would be: 

```javascript
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```